postcss.config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  syntax: 'postcss-scss',
+  parser: 'postcss-safe-parser',
+  plugins: [
+    require('autoprefixer'),
+    require('postcss-import'),
+    require('@fullhuman/postcss-purgecss')({
```

Comment:
@dzole0311 @sandrahoang686 
Here, I am trying to purge out CSS that is not being used. I think we are close based on the preview generated from this PR: https://github.com/NASA-IMPACT/veda-ui/pull/1031 Preview: https://deploy-preview-1031--veda-ui.netlify.app/ 

But this definitely slows down the dev process a lot, so we likely want to include this process only for the build process. And I would like to check what happens when we start customizing scss with tokens.

FYI: I think our use case is very similar to this example from purgecss doc : https://purgecss.com/ant_design.html

Reply:
Nice! Seems like `purgeCss` can't understand nested selectors, while `uswds` makes a heavy use of that pattern. I digged into the `purgecss` issues and [it seems to be the case](https://github.com/FullHuman/purgecss/issues/72)? I pushed a change to safelist some classes and [now it works](https://deploy-preview-1031--veda-ui.netlify.app/), but we should streamline the configuration so that we don't have to add each class manually (looking at the `defaultExtractor` it could be that we can customize it to include nested selectors?).

Reply:
Thanks for looking into the safelist! 
Yes, I agree that manually adding the classes to the whitelist will not be good for the long term. 🤔 I guess we need a 'dynamic safelist'—Making any selector related to the used one not to be purged. I feel like it should be doable, but I can't seem to find a quick way to do it. I will dig a little bit more.

Reply:
Hmm sorry just catching up.. but just reading up on this and other options that we could do. I read up on this article [here](https://www.useanvil.com/blog/engineering/purgecss-styled-components/#feature-comparison-what-about-nested-selectors). Which has a great explanation but then offers a solutions of considering other alternatives other than `purgeCss` - like `purifyCss` https://www.useanvil.com/blog/engineering/purgecss-styled-components/#alternatives-to-purgecss, could we considering using that instead? Looks like there are pros/cons for each but good to know there are other options that can also help with this issue

Reply:
Thanks for the suggestion @sandrahoang686. I did some changes [today](https://github.com/NASA-IMPACT/veda-ui/pull/1029#issuecomment-2217268369), and it seems that we might not need to purge the css (at least for now) 🤔